.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "gitversion.tool": {
```

Comment:
This NuGet package isn't created from a csproj as usual, we use a nuspec. There's no C# code.
Usually, we use the GitVersion MSBuild task to automatically compute the package version during the build, but we can't do that here.

Here, we use the GitVersion .NET tool in a PowerShell script to compute the package version (see Build.ps1)